en/documentation/index.md

```diff
@@ -144,9 +149,9 @@ Here is a list of popular tools used by Rubyists:
 
 ### Further Reading
 
-[Ruby-Doc.org][34] maintains a comprehensive list of English
-documentation sources. There are also plenty of [books about Ruby][35].
-If you have questions about Ruby the [mailing
+[Ruby-Doc.org][34] and [Omniref.com][40] both maintain a comprehensive 
```

Comment:
That's _not_ what is meant in the "further reading" paragraph; rather tutorials, blog posts, ...

I removed the reference to Omniref from that paragraph.